.github/workflows/publish-docker-image.yaml

```diff
@@ -0,0 +1,30 @@
+name: Publish Docker image
+
+# This workflow currently publishes to the following account unless we have an
+# official MacPorts repo on DockerHub
+#
+# DockerHub Repo: arjunsalyan/macports-webapp
+# Email: arjun@macports.org
+# Github: @arjunsalyan
+
+on:
+  push:
+    branches: [ master ]
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    environment: main
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.ARJUNSALYAN_DOCKER_USERNAME }}
+          password: ${{ secrets.ARJUNSALYAN_DOCKER_PASSWORD }}
+      - name: Push to Docker Hub
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: arjunsalyan/macports-webapp:latest
```

Comment:
I'm sorry, I don't know anything about this.

Reply:
It's not about any particular knowledge. At the moment this image is published under
https://hub.docker.com/r/arjunsalyan/macports-webapp
while I believe it would make more sense to publish it under the official macports account on docker hub. We technically don't need to change anything to make the website work.

Reply:
I don't know anything about docker, docker hub, docker images... if we have an account for macports at docker hub I don't know who has access to it. If we don't have one, certainly someone could create one; in that case sharing the credentials with the infra list might be a good idea.